skaffold.yml

```diff
@@ -0,0 +1,62 @@
+apiVersion: skaffold/v1beta11
+kind: Config
+
+profiles:
+- name: development
+  activation:
+  - command: dev
+  patches:
+  - op: replace
+    path: /build/artifacts/0/docker/buildArgs/NODE_ENV
+    value: development
+  - op: replace
+    path: /build/artifacts/0/docker/target
+    value: development
```

Comment:
Interesting point. We actually set NODE_ENV in quite a few places (such as build args) but the impact is essentially nil. Given that we target development/production at build time, switching NODE_ENV to development in the production image would result in errors due to not having installed devDependencies (e.g. `good` for logging).

I'll remove all the NODE_ENV declarations outside of `Dockerfile` to prevent confusion.